model/shared/src/main/scala/explore/model/ObsSummary.scala

```diff
@@ -63,12 +70,24 @@ object ObsSummary {
         }
     }
 
+  private case class CS(id: ConstraintSet.Id)
+
```

Comment:
Maybe there's a better way to parse the nested constraint set -> id?
`"constraintSet": { "id": "c-2"}`
But, this is the best I could come up with.

Reply:
I think it's OK.